web/src/app/admin/admin-alert-counts/AlertCountLineGraph.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react'
+import { Grid, Paper, Typography } from '@mui/material'
+import makeStyles from '@mui/styles/makeStyles/makeStyles'
+import { Theme, useTheme } from '@mui/material/styles'
+import {
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  ResponsiveContainer,
+  LineChart,
+  Line,
+  Legend,
+  Tooltip,
+} from 'recharts'
+
+interface AlertCountLineGraphProps {
+  data: typeof LineChart.defaultProps['data']
+}
+
+const useStyles = makeStyles((theme: Theme) => ({
+  graphContent: {
+    height: '500px',
+    fontFamily: theme.typography.body2.fontFamily,
+  },
+  bar: {
+    '&:hover': {
+      cursor: 'pointer',
+    },
+  },
+}))
+
+export default function AlertCountLineGraph(
+  props: AlertCountLineGraphProps,
+): JSX.Element {
+  const [active, setActive] = useState('')
+  const classes = useStyles()
+  const theme = useTheme()
+
+  const chooseColor = (idx: number): string => {
+    switch (idx) {
+      case 1:
+        return '#3f8e98'
+      case 2:
+        return '#209e90'
+      case 3:
+        return '#41ab74'
+      case 4:
+        return '#79b34a'
+      case 5:
+        return '#b9b218'
+      case 6:
+        return '#ffa600'
+      default:
+        return '#607d8b'
+    }
+  }
+
+  return (
+    <Grid container className={classes.graphContent}>
+      <Grid item xs={12} data-cy='metrics-averages-graph'>
+        <ResponsiveContainer width='100%' height='100%'>
```

Comment:
I've found that using [`AutoSizer`](https://github.com/target/goalert/pull/2481/files#diff-08a7c8fc447ba1006dfeef1e2d54ef027c1fce858d75c9fad4291a99a9796d81R56-R72) installed from `react-virtualized-auto-sizer` works well, because it resizes as the browser width changes